test/localmodeling_tests.jl

```diff
@@ -29,6 +29,42 @@ s_test  = data[N_train:end,1]
     end
 end
 
+@testset "LinearLocalModel" begin
+    @testset "D=$D and τ=$τ" for D ∈ [3,4], τ ∈ [14,15]
+        p = 50
+        method = LinearLocalModel()
+        ntype = FixedMassNeighborhood(3)
+        stepsize = 1
+        s_pred = localmodel_tsp(s_train, D, τ, p;
+        method=method, ntype=ntype, stepsize=stepsize)
+        @test length(s_pred) == p+1
+        @test norm(s_test[1:p+1] - s_pred)/p < 5e-2
+
+        #Repeat with reconstruction given
+        R = Reconstruction(s_train, D, τ)
+        s_pred = localmodel_tsp(R, p; method=method, ntype=ntype, stepsize=stepsize)[:,D]
+        @test length(s_pred) == p+1
+        @test norm(s_test[1:p+1] - s_pred)/p < 5e-2
+    end
+
+    @testset "D=$D and τ=$τ" for D ∈ [3,4], τ ∈ [14,15]
```

Comment:
This test title is not very well descriptive. 
How about making one more outer loop of `@testset` that says which method you are using? If only one of the two linear models fail, you won't be able to immediately tell which one of the 2 from the report.